pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -80,6 +80,61 @@ func NewNodeController(
 	}
 }
 
+// Generates num pod CIDRs that could be assigned to nodes.
+func generateCIDRs(num int) util.StringSet {
+	res := util.NewStringSet()
+	for i := 0; i < num; i++ {
+		// TODO: Make the CIDRs configurable.
+		res.Insert(fmt.Sprintf("10.244.%v.0/24", i))
```

Comment:
(Yes, I opened that 6 weeks ago, but I was playing around with about 4 days ago after recent `.kubeconfig` stuff landed. It's still not in good shape for this usecase, unfortunately, but it's better.)